crates/biome_configuration/src/javascript/mod.rs

```diff
@@ -94,7 +94,7 @@ impl FromStr for JsxRuntime {
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
             "transparent" => Ok(Self::Transparent),
-            "react-classic" | "reactClassic" => Ok(Self::ReactClassic),
```

Comment:
It should be `react-classic`, the correct value. These are values accepted from the CLI, and we don't accept camelCase values.

Reply:
It is a value accepted from [the configuration](https://biomejs.dev/reference/configuration/#javascriptjsxruntime). We don't expose a CLI option for that.

Reply:
But we usually `FromStr` for `Bpaf`. Isn't this the case?

Reply:
Yeah, you are right. Thus, this implementation is dead code. I should remove it.